jinja2_standalone_compiler/__init__.py

```diff
@@ -46,12 +49,14 @@ def main(path, settings=None):
             continue
 
         html_template, _ = os.path.splitext(jinja_template)
-        html_template = '{}.html'.format(html_template)
+        if output_options['remove_double_extension']:
+            html_template, _ = os.path.splitext(html_template)
+        html_template = '{}{}'.format(html_template, output_options['extension'])
```

Comment:
Here extension should default to `.html`, so I guess `output_options.get('extension', '.html)` is better. What do you think?